setup.py

```diff
@@ -193,9 +191,12 @@ def run(self):
 
     assert(now_1 <= now  )
     assert(now   <= now_2)
-    assert(now_2  - now_1 <= 0.01)
+    assert(now_2  - now_1 <= 1.)
 
-except:
+except Exception:
+    e_info = sys.exc_info()
+    print('[WARN] Exception was raised: ', e_info[1].__class__.__name__)
+    traceback.print_exc(file=sys.stdout)
```

Comment:
I would suggest to capture the stack trace in a string and add it as comment into the script created below, so that we can inspect the problem after deployment (during deployment pip will swallow this message if the return code it `0` I believe).